components/lcd_touch/esp_lcd_touch_tt21100/esp_lcd_touch_tt21100.c

```diff
@@ -61,7 +69,10 @@ esp_err_t esp_lcd_touch_new_i2c_tt21100(const esp_lcd_panel_io_handle_t io, cons
     esp_lcd_touch_tt21100->get_button_state = esp_lcd_touch_tt21100_get_button_state;
 #endif
     esp_lcd_touch_tt21100->del = esp_lcd_touch_tt21100_del;
-
+#if CONFIG_ESP_LCD_TOUCH_SLEEP_SUPPORT
```

Comment:
Same as in `esp_lcd_touch` - remove all ifdefs `CONFIG_ESP_LCD_TOUCH_SLEEP_SUPPORT`.

Reply:
Hello
`esp_lcd_touch_handle_t` is defined in `esp_lcd_touch`, I added this macro because I was worried that the two components would not match.

You mean，`esp_lcd_touch_tt21100` relys on a component that supports low-power interfaces, and delete all the `CONFIG_ESP_LCD_TOUCH_SLEEP_SUPPORT` in both components (esp_lcd_touch_tt21100 & esp_lcd_touch)?

Reply:
Yes, I thing that we don't need this macro in both components (I wrote it in #212 too). It is same as another functions.